README.md

```diff
@@ -90,6 +90,66 @@ docker build -t pcengines/pce-fw-builder -f Dockerfile.ml .
 ```
 docker build -t pcengines/pce-fw-builder-legacy -f Dockerfile.legacy .
 ```
+Building Coreboot with Tianocore payload
+---------------------
+
+In this example coreboot v4.11.0.6 is build for apu2 platform, 
+but release version and platform can be changed.
+
+1. Clone the pce-fw-builder
+
+2. Pull or build docker container:
+
+
+3. Build image.
+
+  ```
+  ./build.sh release v4.11.0.6 apu2
+  ```
+
+4. Invoke distclean:
+
+  ```
+  ./build.sh dev-build $PWD/release/coreboot apu2 distclean
+  ```
+
+5. Copy config file for target platform
+
+  ```
+  cp $PWD/release/coreboot/configs/config.pcengines_apu2 $PWD/release/coreboot/.config
+  ```
+
+6. Create full config:
+
+  ```
+  ./build.sh dev-build $PWD/release/coreboot apu2 olddefconfig
+  ```
+
+7. Invoke menuconfig:
+
+  ```
+  ./build.sh dev-build $PWD/release/coreboot apu2 menuconfig
+  ```
+
+8. In menuconfig go to `Payload` menu and next:
+
+  - In `Add a payload` choose *Tianocore coreboot payload package*
+  - Select tianocore build type release
+  - In `Secondary Payloads` disable all options
+  - Make sure that `Insert a commit’s SHA-1 or a branch name` 
+  is set to `origin/coreboot-4.7.x-uefi`
```

Comment:
Please add information that it is the Tianocore revision setting. Additionally, you could add a screenshot from Payloads menu?